### PR TITLE
Maternal disorders YLD equation fix

### PR DESCRIPTION
**Pull request by alibow**
Found a small error in our YLD equation for maternal disorders from the IV iron implementation. Updated the equation in the 2021 document (and noted the updated from the prior implementation) and called out that the implemented equation in the 2019 document is incorrect but kept it there for reference.

### CODE REVIEW

**Review by aflaxman — COMMENTED** [review]
I'm confused, let's talk this through in real time.

File: docs/source/models/causes/maternal_disorders/gbd_2019/index.rst [hunk 1]
```diff
@@ -312,6 +312,19 @@ Years lived with disability
...
+.. note::
+
+  The equation shown below is inconsistent with the approach we implemented of accumulating all maternal disorders YLDs in a single timestep rather than over the course of 1 year. With maternal disorders YLDs accumulated in a single timestep, the equation should be: 
+
+  .. math::
+
+    \text{YLDs per non-fatal maternal disorders case} = 
+    
+
+    \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - csmr_\text{c366}}
```
Comment (aflaxman) [inline docs/source/models/causes/maternal_disorders/gbd_2019/index.rst:324]:
```suggestion
    \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - \text{csmr}_\text{c366}}
```

File: docs/source/models/causes/maternal_disorders/gbd_2021/index.rst [hunk 2]
```diff
@@ -342,23 +341,21 @@ Simulants who experience an incident case of maternal disorders and occupy the p
   \text{YLDs per non-fatal maternal disorders case} = 
   
 
-  \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - (ACMR - csmr_\text{c366}) * \text{incidence_rate}_\text{c366} - csmr_\text{c366}}
+  \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - csmr_\text{c366}}
```
Comment (aflaxman) [inline docs/source/models/causes/maternal_disorders/gbd_2021/index.rst:344]:
```suggestion
  \frac{\text{ylds}_{c366} - \text{ylds}_\text{s182,s183,s184}}{\text{incidence_rate}_{c366} - \text{csmr}_\text{c366}}
```

**Review by aflaxman — APPROVED** [review]
Ali explained it to me and it all makes sense now.

**Review by patricktnast — APPROVED** [review]

**Review by SylLutze — APPROVED** [review]

**Pull request merged**